components/TodoItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet,Text,  } from "react-native";
+
+
+
+const todoItem = (props) => {
```

Comment:
Please add typescript annotation to props. See my components in the masjid app web repo.